docs/docs/share-theme.mdx

```diff
@@ -39,7 +39,7 @@ There are a couple of parameters you can use to tweak the image rendering:
 </TabItem>
 <TabItem value="others">
 
-The oh-my-posh executable has the `--export-png` switch to export your current theme configuration.
+The oh-my-posh executable has the `--export-png` switch to export your current theme configuration to the current directory location.
```

Comment:
I would omit `location` as a directory already is a location (for me that is).

Reply:
Makes sense to me!